irods_consortium_continuous_integration_test_hook.py

```diff
@@ -34,7 +34,14 @@ def get_package_type():
 def install_test_prerequisites():
     irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', '--upgrade', 'pip>=20.3.4'], check_rc=True)
     irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', 'boto3', '--upgrade'], check_rc=True)
-    irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', 'minio', '--upgrade'], check_rc=True)
+
+    # Minio 7.1.17 imports the annontations module which only exists in Python 3.7 and beyond.
+    # CentOS 7 defaults to Python 3.6, therefore we have to install the previous version of Minio
```

Comment:
Is this also the case for Ubuntu 18.04?

Reply:
If Ubuntu 18 ships with Python 3.6, then yes.

We'll hold on merging until we confirm.

Reply:
This also affects Almalinux 8.

Reply:
Definitely. (b) looks like the best approach. We know this issue is specific to python 3.6.

Reply:
Done.